src/main/java/green/citibike/aws/Response.java

```diff
@@ -0,0 +1,91 @@
+package green.citibike.aws;
+
+import green.citibike.json.StationInfo;
+import green.citibike.json.StationStatus;
+
+public class Response {
+    private From from;
+    private Start start;
+    private To to;
+    private End end;
+
+    public Response(From from, Start start, To to, End end) {
+        this.from = from;
+        this.start = start;
+        this.to = to;
+        this.end = end;
+    }
+
+    public Response(StationStatus from, StationStatus to) {
+        this.from = new From(from.getLat(), from.getLon());
+        this.start = new Start(from.getLat(), from.getLon(), from.getName(), from.getStationId());
+        this.to = new To(to.getLat(), to.getLon());
+        this.end = new End(to.getLat(), to.getLon(), to.getName(), to.getStationId());
+    }
+
+    public From getFrom() {
+        return from;
+    }
+
+    public To getTo() {
+        return to;
+    }
+
+    public Start getStart() {
+        return start;
+    }
+
+    public End getEnd() {
+        return end;
+    }
+
+    public class From {
```

Comment:
Doesn't it seem a little wasteful that From/To are the same and Start/End are the same?

Reply:
Yes!! That's why I didn't do it the first time but if it's json it has to match all the names...?

Reply:
The variable names need to match.
**Not the class names.**
You can use the same class for From/To and Start/End.